packages/core/test/invoke.test.ts

```diff
@@ -2632,6 +2632,31 @@ describe('invoke', () => {
       interpret(machine).start();
     });
   });
+
+  it('invoke creator should provide reference to self (invokee)', (done) => {
```

Comment:
Isn't the child actor an invokee here? 
```suggestion
  it('invoke creator should provide reference to self', (done) => {
```